tests/testthat/test-add.R

```diff
@@ -164,6 +164,21 @@ test_that("can recycle when adding columns", {
   expect_identical(df_new$c, 3:1)
 })
 
+test_that("can recycle when adding a column of length 1", {
+  df <- tibble(a = 1:3)
+  df_new <- add_column(df, b = 4)
+  expect_identical(ncol(df_new), ncol(df) + 1L)
+  expect_identical(df_new$b, rep(4, 3))
+})
```

Comment:
Could you explain why we test with explicit details like this instead of the shorter `expect_identical(df_new, tibble(a = 1:3, b = rep(4, 3))`?

For consistency I used the current way of testing, but curious why.